ucsdscheduleme/Models/RateMyProfessor.cs

```diff
@@ -9,9 +9,9 @@ namespace ucsdscheduleme.Models
     public class RateMyProfessor
     {
         public int Id { get; set; }
-        public decimal OverallQuality { get; set; }
```

Comment:
I feel like it would be helpful to specific the range for each of these quantities; like 1-5